server/modules/activityLog/api/activityLogApi.js

```diff
@@ -0,0 +1,20 @@
+import * as Request from '@server/utils/request'
+import * as AuthMiddleware from '../../auth/authApiMiddleware'
+import * as ActivityLogService from '../service/activityLogService'
+
+export const init = app => {
+  // ==== READ
+
+  app.get(`/survey/:surveyId/activity-log`, AuthMiddleware.requireSurveyEditPermission, async (req, res, next) => {
+    try {
+      const { surveyId, offset } = Request.getParams(req)
```

Comment:
I would pass also limit as rest parameter

Reply:
I didn't pass it to avoid to check that parameter; should it be always required and limited to 100 for example?

Reply:
you need the limit in the UI to add the placeholder to incrementally fetch the next sequence of items as we discussed earlier, that's why i'm suggesting you add it now

Reply:
but the limit in the query is always the same, I think you will pass different offsets every time... right?